src/common/metric.tsx

```diff
@@ -17,14 +20,27 @@ export const ALL_METRICS = [
   Metric.POSITIVE_TESTS,
   Metric.HOSPITAL_USAGE,
   Metric.VACCINATIONS,
+
+  // TODO(8.2)
```

Comment:
What is left to do here?

Reply:
oh whoops. left too many unnecessary TODOs